helm/aws-admission-controller/templates/deployment.yaml

```diff
@@ -32,34 +32,32 @@ spec:
         runAsGroup: 1000
       containers:
         - name: {{ include "name" . }}
-          image: "{{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+          image: "{{ .Values.registry.domain }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
           env:
             - name: DEFAULT_AWS_AZS
-              value: {{ join "," .Values.Installation.V1.Provider.AWS.AvailabilityZones }}
+              value: {{ join "," .Values.aws.availabilityZones }}
             - name: DEFAULT_DOCKER_CIDR
-              value: {{ .Values.Installation.V1.Guest.Docker.CIDR }}
+              value: {{ .Values.workloadCluster.docker.cidr }}
             - name: DEFAULT_IPAM_NETWORKCIDR
-              value: {{ .Values.Installation.V1.Guest.IPAM.NetworkCIDR }}
+              value: {{ .Values.workloadCluster.ipam.cidr }}
             - name: DEFAULT_KUBERNETES_ADMIN_GROUP
-              value: {{ .Values.Installation.V1.Kubernetes.Auth.TenantAdminTargetGroup }}
-            - name: DEFAULT_KUBERNETES_ALL_GROUP
-              value: {{ .Values.Installation.V1.Kubernetes.Auth.ViewAllTargetGroup }}
+              value: {{ .Values.managementCluster.kubernetes.auth.tenantAdminTargetGroup }}
             - name: DEFAULT_KUBERNETES_CLUSTER_IP_RANGE
-              value: {{ .Values.Installation.V1.Guest.Kubernetes.API.ClusterIPRange }}
-            - name: DEFAULT_KUBERNETES_ENDPOINT              
-              value: {{ .Values.Installation.V1.Guest.Kubernetes.API.EndpointBase }}
+              value: {{ .Values.workloadCluster.kubernetes.api.clusterIPRange }}
+            - name: DEFAULT_KUBERNETES_ENDPOINT
+              value: {{ .Values.workloadCluster.baseDomain }}
             - name: DEFAULT_AWS_INSTANCE_TYPES
-              value: {{ join "," .Values.Installation.V1.Provider.AWS.EC2.Instance.Allowed }}
+              value: {{ join "," .Values.aws.instance.allowed }}
             - name: DEFAULT_AWS_REGION
-              value: {{ .Values.Installation.V1.Provider.AWS.Region }}
+              value: {{ .Values.aws.region }}
+            {{- $workloadClusterCNICIDRSplit := splitn "/" 2 .Values.workloadCluster.cni.cidr }}
```

Comment:
why is the split here?

Reply:
is it just to save variables and have 2 values in one?

Reply:
Yeah, this `.Values.Installation.V1.Guest.Calico.CIDR` and `.Values.Installation.V1.Guest.Calico.Subnet` makes zero sense to me. Now you have `.Values.workloadCluster.cni.cidr` in values.yaml. It's just internally weird, but I don't want to deal with that. You can address that any time if you want to.